src/components/Alphabet/Letter/Letter.js

```diff
@@ -1,34 +1,62 @@
 import React from 'react';
 import styles from './Letter.css';
+import classnames from 'classnames';
 
-const checkLetter = (letter, id, onLetterClick) => {
-    const url = `/phrases/check`;
-    return fetch(url, {
-        headers: {
-            'Content-Type': 'application/json',
-            'Accept': 'application/json',
-        },
-        method: 'post',
-        body: JSON.stringify({
-            id,
-            letter,
+class Letter extends React.PureComponent {
+    state = {
+        isLetterCorrect: null,
+        usedLetter: false,
```

Comment:
Should maybe be 'wasUsed' instead of 'usedLetter'
Also if the letter was correct or not, you could get that information from that, instead of having separate state variable.

Reply:
We came to the conclusion that this variable is unnecessary.